flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
so you'll never test it. as for poor floating point.... this is not what I have seen.

Reply:
I managed to build for aarch64 without OpenBLAS on GitHub Actions. With OpenBLAS the build times out after 5 hours.